Hexastore.Web/Controllers/UpdateRequest.cs

```diff
@@ -13,6 +13,6 @@ public class UpdateRequest
         public string PartitionId { get; set; }
 
         [JsonProperty("data")]
-        public JToken Data { get; set; }
+        public JObject Data { get; set; }
```

Comment:
More just curious: why the switch from token to object here?

Reply:
Initially kept it JToken because it could be an array or an object. But with the support for arrays at the controller level this should always be an individual object and not a batch. So JObject makes more sense.